packages/clerk-js/src/ui/utils/createSlug.ts

```diff
@@ -0,0 +1,2089 @@
+const escapeStringRegexp = (string: string): string =>
```

Comment:
Does FAPI also check for a valid slug?

Reply:
@nikosdouvlis yes it does, but we need to do a check each time the user types a character so they can have real-time feedback on the result